lib/tasks/demo.rake

```diff
@@ -107,17 +167,22 @@ namespace :demo do
       end
     end
 
+    def reviewer
```

Comment:
can we be sure we already have reviewers? or should this be

    User.where(permission_id: 3).first || FactoryGirl.create(:reviewer)